examples/mars.html

```diff
@@ -0,0 +1,90 @@
+<html>
+    <head>
+        <title>Itowns - Mars</title>
+
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="css/example.css">
+        <link rel="stylesheet" type="text/css" href="css/LoadingScreen.css">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.6/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <span id="divScaleWidget"> Scale </span>
+        <div id="miniDiv"></div>
+
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="js/GUI/LoadingScreen.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            // # Simple Globe viewer
+            // Define initial camera position
+            itowns.proj4.defs('EPSG:104905', '+proj=longlat +a=3396190 +rf=169.894447223612 +no_defs +type=crs');
+            var placement = {
+                coord: new itowns.Coordinates('EPSG:4326', 77.6, 18.2),
+                range: 25000000,
+            }
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            // Instanciate iTowns GlobeView*
+            var view = new itowns.GlobeView(viewerDiv, placement, {
+                atmosphere: {
+                    Kr: 0.0025,
+                    Km: 0.0015,
+                    ESun: 15.0,
+                    g: -0.950,
+                    innerRadius: 6400000,
+                    outerRadius: 6600000,
+                    wavelength: [0.350, 0.470, 0.575],
+                    scaleDepth: 0.38,
+                }
+            });
+            setupLoadingScreen(viewerDiv, view);
+
+            // Add a tms imagery source
+            var tmsMarsSource = new itowns.TMSSource({
+                    // crs: 'EPSG:104905',
```

Comment:
Could this comment be removed as well as the same on line 69 ? If so, I believe the definition of `EPSG:104905` on line 24 could also be removed, since I don't see it used anywhere else.

Reply:
I will add a comment the mars projection is `EPSG:104905` but i use earth to add mars color layer...